WordPress/Classes/ViewRelated/NUX/CreateAccountAndBlogViewController.m

```diff
@@ -699,8 +699,6 @@ - (void)createUserAndSite
     WPAsyncBlockOperation *userCreation = [WPAsyncBlockOperation operationWithBlock:^(WPAsyncBlockOperation *operation){
         void (^createUserSuccess)(id) = ^(id responseObject){
             // Turn on the new editor only for users that create a new account within the iOS app
-            [WPPostViewController setNewEditorAvailable:YES];
-            [WPPostViewController setNewEditorEnabled:YES];
             [WPAnalytics track:WPAnalyticsStatEditorEnabledNewVersion];
```

Comment:
@bummytime - I fixed the tracking as you proposed.  Good call!  Please let me know if there's anything else off.